connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSourceTask.java

```diff
@@ -363,6 +363,7 @@ private boolean sendRecords() {
             try {
                 maybeCreateTopic(record.topic());
                 final String topic = producerRecord.topic();
+                log.debug("{} is going to send record to {}", WorkerSourceTask.this, topic);
```

Comment:
Writing one log message per record, seems excessive, even for `DEBUG`

There's a trace level message above for the record. Maybe we want to append the topic name there.

Reply:
OK